sdk/monitor/azure-monitor-query/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Bugs Fixed
 
 ### Other Changes
+- Migrate test recordings to assets repo.
```

Comment:
We don't need to mention this here as it's not a change that customers need to know about.
```suggestion
```